### DISCUSSION TITLE
How to raise error in Javascripts Katas?

### DISCUSSION THREAD

**Discussion by jotakar** [codewars/codewars.com#2433] [Q&A] [answered]
Some katas ask for throw an error in some situations, but I've tried to use try...catch, and trow new Error() and it doesn' work. And I van not find any help in docs. So how can I raise errors in katas? (Javascript)

**Comment by DonaldKellett** [comment]
Maybe try using `throw new Error('some error message')` without try / catch
blocks. You just have to throw the error from within the function; the
tests will catch it accordingly (as long as the thrown error is expected
from the tests).

**Reply by jotakar** [reply]
Finally I got it. I wrote throw true, without try catch.

**Comment by hobovsky** [comment, marked as answer]
The way of signaling an error might depend on a kata, so it would be better to ask in discourse of a specific kata.

What seems suspicious is that a kata asks you to _raise_ an error, but for some reason you went with try/catch what is an exact opposite of the task (`try` and `catch` are used to _handle_ an error, and not to raise it). `try` and `catch` should not be necessary in solution (but again, it depends on what exactly kata wants you to do).
